src/view/configuration/configuration.jsx

```diff
@@ -44,7 +44,7 @@ const contextOptions = ["web", "device", "environment", "placeContext"];
 
 const getInstanceDefaults = initInfo => ({
   name: "alloy",
-  propertyId: "",
+  configId: "",
   imsOrgId: initInfo.company.orgId,
   edgeDomain: "alpha.konductor.adobedc.net",
```

Comment:
We are currently hitting `beta.adobedc.net`. Should we update it here now, or wait for the 3rd party domain changes that are coming?

Reply:
Changing it here would be good.